src/jx/reporter/karma.cljs

```diff
@@ -62,6 +63,7 @@
     (karma-result! result)))
 
 (defmethod cljs.test/report [::karma :fail] [m]
+  (println m)
```

Comment:
this seems to contain correct map now (https://github.com/honzabrecka/karma-reporter/pull/8#issuecomment-262061641)